contracts/contracts/mocks/MockUniswapRouter.sol

```diff
@@ -48,6 +58,14 @@ contract MockUniswapRouter is IUniswapV2Router {
         payable
         returns (uint256 amountOut)
     {
+        bytes memory tok0Bytes = new bytes(20);
+        for (uint256 i = 0; i < 20; i++) {
+            tok0Bytes[i] = params.path[i];
```

Comment:
There are way faster / cleaner ways to do this, but they take assembly language. This is good enough for a mock.

Reply:
Yeah it is messy, glued together with duct tape :)